bitcoin/src/address.rs

```diff
@@ -672,7 +738,7 @@ impl Address {
 
```

Comment:
I think `is_standar` should be allowed on unchecked.

Reply:
I could go either way, but my inclination is no ... if you don't know what network an address is for you shouldn't be able to figure out "what kind of address it is" any more precisely. But I dunno.

Reply:
So far I kept it only on valid. I believe it would be easier to add it to unchecked, if needed in the future, than removing it.

Reply:
OK, let's stay conservative for now.